### PR TITLE
Evolve[Doc]: Resolve unversioned version numbers

### PR DESCRIPTION
**Pull request by taqtiqa-mark**
The existing Versioned and Unversioned adjectives would lead to this type of sentence:

> Unversioned migration version numbers are restricted to `u32` (unsigned, 32-bit integers).

This seems less contradictory.

### CODE REVIEW

**Comment by jxs** [comment]
Hi, and  thanks! Can you resolve the conflicts?

File: README.md [hunk 1]
```diff
@@ -20,21 +20,22 @@ Powerful SQL migration toolkit for Rust.
...
-If you are using a driver that is not yet supported, namely [`SQLx`](https://github.com/launchbadge/sqlx) you can run migrations providing a [`Config`](https://docs.rs/refinery/latest/refinery/config/struct.Config.html) instead of the connection type, as `Config` impl's `Migrate`. You will still need to provide the `postgres`/`mysql`/`rusqlite`/`tiberius` driver as a feature for [`Runner::run`](https://docs.rs/refinery/latest/refinery/struct.Runner.html#method.run) and `tokio-postgres`/`mysql_async` for [`Runner::run_async`](https://docs.rs/refinery/latest/refinery/struct.Runner.html#method.run_async).\
-`refinery` works best with [`Barrel`](https://crates.io/crates/barrel) but you can also have your migrations in .sql files or use any other Rust crate for schema generation.
+Refinery strives to make running migrations for different databases as easy as possible.
+It works by running your migrations on a provided database connection, either by embedding them on your Rust code, or via the `refinery` CLI.
```
Comment (jxs) [inline README.md:24]: the link for `[refinery_cli]` from the last commit should be maintained

**Review by jxs — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jxs) [reply README.md:24]: Hi, I see you resolved this conversation, but the issue remains to be addressed:
there was previously a link to refinery_cli `[refinery_cli]` with the link that still exists at line 115

Comment (taqtiqa-mark) [reply README.md:24]: Apologies, I believe the commit below addresses the comment?

Comment (jxs) [inline README.md:24]:
```suggestion
It works by running your migrations on a provided database connection, either by embedding them on your Rust code, or via the [refinery_cli].
```

File: README.md [hunk 2]
```diff
@@ -20,21 +20,22 @@ Powerful SQL migration toolkit for Rust.
...
+`refinery` works best with [`Barrel`](https://crates.io/crates/barrel) but you can also have your migrations in `.sql` files or use any other Rust crate for schema generation.
 
 ## Usage
 
 - Add refinery to your Cargo.toml dependencies with the selected driver as feature eg: `refinery = { version = "0.8", features = ["rusqlite"]}`
 - Migrations can be defined in .sql files or Rust modules that must have a function called `migration` that returns a [`String`](https://doc.rust-lang.org/std/string/struct.String.html).
 - Migrations can be strictly versioned by prefixing the file with `V` or not strictly versioned by prefixing the file with `U`.
 - Migrations, both .sql files and Rust modules must be named in the format `[U|V]{1}__{2}.sql` or `[U|V]{1}__{2}.rs`, where `{1}` represents the migration version and `{2}` the name.
-- Migrations can be run either by embedding them in your Rust code with `embed_migrations` macro, or via [refinery_cli].
+- Migrations can be run either by embedding them in your Rust code with `embed_migrations` macro, or via [`refinery`](refinery_cli) CLI`.
```
Comment (jxs) [inline README.md:36]:
```suggestion
- Migrations can be run either by embedding them in your Rust code with `embed_migrations` macro, or via [refinery_cli].
```

**Comment by jxs** [comment]
thanks!

**Pull request merged**